pgrx-pg-sys/src/submodules/transaction_id.rs

```diff
@@ -0,0 +1,74 @@
+//LICENSE Portions Copyright 2019-2021 ZomboDB, LLC.
+//LICENSE
+//LICENSE Portions Copyright 2021-2023 Technology Concepts & Design, Inc.
+//LICENSE
+//LICENSE Portions Copyright 2023-2023 PgCentral Foundation, Inc. <contact@pgcentral.org>
+//LICENSE
+//LICENSE All rights reserved.
+//LICENSE
+//LICENSE Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+use pgrx_sql_entity_graph::metadata::{
+    ArgumentError, Returns, ReturnsError, SqlMapping, SqlTranslatable,
+};
+
+pub type MultiXactId = TransactionId;
+
+/// An `xid` type from PostgreSQL
+#[repr(C)]
```

Comment:
I think we want this to be `#[repr(transparent)]`.  

The reasons being that it communicates to the user that our `struct TransactionId` is wrapping some other type and it will also cause some friction for anyone that tries to PR changing this struct by adding another field.